components/content/index.css

```
@@ -1,21 +1,42 @@
 .content {
     background-color: var(--blueviolet);
+    border-radius: 12px;
 }
 
 .content-holder {
     display: flex;
     height: 100%;
-    background-image: linear-gradient(var(--red), var(--green));
+    border-radius: 12px;
```

```suggestion
    border-radius: 12px;
    overflow: hidden;
```

Your border-radius isn't working because children overflow parents and you cannot see changes. Use `overflow: hidden;`